### ISSUE TITLE
Mobile Apps for paperless?

### ISSUE THREAD

**Issue by ZyanKLee** [the-paperless-project/paperless#580]
Hello there,

First of all: thanks for your great work on this project.

I would like to relay a question my wife had:
Are there any known-of Android apps for paperless? Or is the UI responsive and capable of using the mobile's camera to add documents?

**Comment by CkuT** [comment]
Not to my knowledge. Personally I am using [Notebloc](https://play.google.com/store/apps/details?id=com.notebloc.app&hl=fr) to handle my documents. Notebloc can send cropped photos by email, which [can be retrieved](https://paperless.readthedocs.io/en/latest/consumption.html#imap-email) by paperless.

However it is a nice idea, a mobile app could be great !

**Comment by Pyromane** [comment]
Recently there seems to be something:  #642

**Comment by bauerj** [comment]
Yes, thank you for mentioning it @Pyromane! However, I'm not sure if this is what @ZyanKLee wanted (yet). Currently it's only possible to access existing documents. Also, dedicated scanning apps like the aforementioned Notebloc, Adobe Scan or Microsoft Office Lens will probably produce a better result and are already compatible with Paperless.
